pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -50,17 +50,6 @@ in stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" "test" ];
 
-  patches = [
-    # https://github.com/bluez/bluez/commit/0905a06410d4a5189f0be81e25eb3c3e8a2199c5
-    # which fixes https://github.com/bluez/bluez/issues/329
-    # and is already merged upstream and not yet in a release.
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Apologies for that. Here you go -> https://github.com/NixOS/nixpkgs/pull/187392